docs/tutorial.md

```diff
@@ -449,7 +439,7 @@ kubectl create clusterrolebinding tutorial-binding \
 ```
 
 
-To run the `Pipeline`, create a [`PipelineRun`](pipelineruns.md) as follows:
+To run your `Pipeline`, instantiate it with a [`PipelineRun`](pipelineruns.md) as follows:
```

Comment:
Should be future tense as user won't create PipelineRun here:

```suggestion
To run your `Pipeline`, you will create it with a [`PipelineRun`](pipelineruns.md) that is defined below:
```

Reply:
Original statement conveys the same meaning.